src/backend/src/utils/auth.utils.ts

```diff
@@ -31,7 +31,8 @@ export const requireJwtProd = (req: Request, res: Response, next: NextFunction)
   if (
     req.path === '/users/auth/login' || // logins dont have cookies yet
     req.path === '/' || // base route is available so aws can listen and check the health
-    req.method === 'OPTIONS' // this is a pre-flight request and those don't send cookies
+    req.method === 'OPTIONS' || // this is a pre-flight request and those don't send cookies
+    req.path === '/slack' // slack http endpoint is only used from slack api
```

Comment:
we should have some sort of authentication for slack requests as well, like how we hvae the notification endpoint auth

Reply:
are we able to configure a secret to send along with our slackbots requests?

Reply:
the package we use to handle the endpoint does all the authentication automatically

Reply:
Yeah the api has a built in validation system that invloves sending a challenge parameter to ensure the request is from slack

Reply:
oh really. so what happens if i try to ping /slack from postman it just fails automatically? thats quite nice

Reply:
however dont we want tot make sure that its our slackbot thats pinging it. What if somebody elses slack bot tries to reach this endpoint, we shouldn’t allow those connections

Reply:
We initialize the listener with our signing secret which is then used to authorize events, so if someone else's slack bot tries to reach the endpoint they would have a different signing secret and the authorization would fail

Reply:
oh ok sweet.